libs/net/core/Extensions/DateTimeExtensions.cs

```diff
@@ -14,6 +14,7 @@ public static class DateTimeExtensions
     public static DateTime ToTimeZone(this DateTime date, string timeZoneId)
     {
         var timezone = TimeZoneInfo.FindSystemTimeZoneById(timeZoneId);
-        return TimeZoneInfo.ConvertTime(date, timezone);
+        var result = TimeZoneInfo.ConvertTime(date, timezone);
```

Comment:
We now make the time local after applying a timezone.  This isn't necessary, but it makes it easier to debug/read.